internal/pb/fs.proto

```diff
@@ -64,6 +66,7 @@ message ObjectQuery {
     bool is_prefix = 2;
     bool with_content = 3;
     repeated string ignores = 4;
+    optional bool with_hash = 5;
```

Comment:
I went with an optional here. Other option I think is to make this required and handle current calls with a default value of `false` in the client?